modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/frontend/taglib/servlet/taglib/UserPasswordScreenNavigationEntry.java

```diff
@@ -63,6 +63,10 @@ public boolean isVisible(User user, User selUser) {
 		try {
 			PasswordPolicy passwordPolicy = selUser.getPasswordPolicy();
 
+			if (passwordPolicy == null) {
+				return true;
```

Comment:
Hi @oliverkecskemety . Since it is a regression bug caused by LPS-179689, I noticed that the[ previous code](https://github.com/liferay-appsec/liferay-portal/commit/32afe309c7ecb226a891d8bb1a0699bc2a926bda) was returning "true" when the password policy was not being considered in the first place. That said, if the user doesn't have a password policy, it will be visible. 

About the change in the catch block, I just commited it. 
Please let me know if further modification is needed. :grin: 

Thank you very much

Reply:
Good observation, we came to the same conclusion! Thanks for the change I just wanted to have consistent error handling.